tests/phpunit/elementor/core/base/test-base-object.php

```diff
@@ -10,12 +10,23 @@ class Test_Base_Object extends Elementor_Test_Base {
 	public function test_has_own_method() {
 		$child_class = new Has_Own_Method_Child_Test_Class();
 
+		// Tests with only 1 parameter (method name) passed to has_own_method().
 		$this->assertTrue( $child_class->has_own_method( 'uninherited_method' ) );
 		$this->assertFalse( $child_class->has_own_method( 'inherited_method' ) );
+
+		// Tests with both parameters (method name and base class name) passed to has_own_method().
+		$this->assertTrue( $child_class->has_own_method( 'inherited_from_grandparent_method', Has_Own_Method_Parent_Test_Class::class ) );
```

Comment:
It might be counter intuitive.
`has_own_method()` with one parameter returns true if the method was declared in the class that called `has_own_method()`. 

In this test, the calling class is `Has_Own_Method_Child_Test_Class`.

If a child class calls `has_own_method()` with two parameters, it would return `true` only if its **declaring class** is the class passed as a second parameter. Otherwise, it returns false.